File: lista.segunda.un.vet.34.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
  const int tam = 10;
  vector<int> vetor;
  int numero;

  cout << "Digite 10 numeros diferentes: " << endl;
  while (vetor.size() < tam) {
    cin >> numero;

    bool existe = false;
    for (int i = 0; i < vetor.size(); i++) {
      if (vetor[i] == numero) {
        existe = true;
        break;
      }
    }

    if (existe) {
      cout << "digite um número diferente: " << endl;
    } else {
      vetor.push_back(numero);
    }
  }

  cout << "Vetor: ";
  for (int i = 0; i < vetor.size(); i++) {
    cout << vetor[i] << " ";
  }
  cout << endl;

  return 0;
}

File: lista.segunda.un.vet.34.py
def main():
    tam = 10
    vetor = []

    print("Digite 10 numeros diferentes:")
    while len(vetor) < tam:
        numero = int(input())

        if numero in vetor:
            print("Digite um número diferente:")
        else:
            vetor.append(numero)

    print("Vetor:", " ".join(map(str, vetor)))


if __name__ == "__main__":
    main()
